### PR TITLE
feat: emotion package

### PR DESCRIPTION
**Pull request by AmandaKr**
Added emotion package and theme to frontend and design-system. 
@imklau

### CODE REVIEW

File: packages/frontend/app/pages/_app.js [hunk 1]
```diff
@@ -4,6 +4,8 @@ import "../assets/css/style.css"
 import { createContext } from "react"
 import { fetchAPI } from "../lib/api"
 import { getStrapiMedia } from "../lib/media"
+import { ThemeProvider } from "@emotion/react"
+import theme from "../components/theme"
```
Comment (im-klau) [inline packages/frontend/app/pages/_app.js:8]: use import from the `design-system` directory and remove this additional theme 🙏

File: packages/design-system/package.json [hunk 2]
```diff
@@ -15,7 +15,9 @@
     "@storybook/testing-library": "^0.0.13",
     "babel-loader": "^8.2.5"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@emotion/react": "^11.10.4"
```
Comment (im-klau) [inline packages/design-system/package.json:19]: could you also add the emotion package to the `frontend/app` directory? We'll probably need it there too.

File: packages/frontend/app/pages/_app.js (hunk 1 above)
Comment (AmandaKr) [reply packages/frontend/app/pages/_app.js:8]: Actually I did it, but I don't know why this import stayed here, anyway I will try to fix it

**Review by im-klau — APPROVED** [review]

**Pull request merged**